projects/dsp-ui/src/lib/action/components/message/message.component.ts

```diff
@@ -0,0 +1,208 @@
+import { Location } from '@angular/common';
+import { Component, Input, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { StatusMsg } from '../../../assets/i18n/statusMsg';
+
+/**
+ * @ignore
+ * Data type for messages
+ */
+export class DspMessageData {
+    status: number;
+    statusMsg?: string;
+    statusText?: string;
+    type?: string;
+    route?: string;
+    footnote?: string;
+    errorInfo?: string;
+    url?: string;
+}
+
+@Component({
+  selector: 'dsp-message',
+  templateUrl: './message.component.html',
+  styleUrls: ['./message.component.scss']
+})
+export class MessageComponent implements OnInit {
+
+    /**
+     * Message type: DspMessageData or ApiServiceError
+     *
+     * @param message This type needs at least a status number (0-511).
+     * In this case, or if type is ApiServiceError, it takes the default status messages
+     * from the list of HTTP status codes (https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)
+     */
+    @Input() message: DspMessageData = new DspMessageData();
+
+    /**
+     * @param short Show short message only
+     * It can be used in form to show if a post was successfull or not.
+     */
+    @Input() short = false;
+
+    /**
+     * @param medium Show medium message
+     * A message box without footnote or links.
+     */
+    @Input() medium = false;
```

Comment:
I think that's smart! Otherwise someone could use both parameters and I'm not sure how the system would handle that. For the names, how about: "short", "medium", "long"?

Reply:
added TODO comment in dc48a9e